docs/manual/common/concepts/InternalAndExternalCommunication.md

```diff
@@ -0,0 +1,7 @@
+#Internal and external communication
+
+Lagom promotes use of asynchronous communication without preventing use of synchronous communication where necessary. For example, as mentioned in [[Polyglot systems with Lagom|PolyglotSystems]], legacy application components might use synchronous communication that cannot be changed.  When choosing the mode of communication for your microservices, keep in mind inter-service communication as well as external communication. 
```

Comment:
I'd prefer to leave the page in as a placeholder. I've removed the last sentence.